Pack.java

```diff
@@ -0,0 +1,28 @@
+package Pack;
+
+public class Pack {
+    public static void main(String[] args) {
+        int color = 2;
+
+        if (color == 1) {
+            System.out.println("Красный");
+        }
+        if (color == 2) {

```

Comment:
It's necessary to use if-else if-else here, not separate if.
Because if we have more complex condition, it could ocur, that we can 'visit' not only one block